mkweights/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we want to include the yarn lock file? At least for jekyll, I usually don't add the Gemfile.lock because it adds more issues than helps.

Reply:
Most of the time for node projects yes you should. Because the `package.json` dependencies allow you to install newer semver feature or patch updates at random.
```
  "dependencies": {
    "toml": "^3.0.0"
  }
```
(`^3.0.0` matches toml 3.1.3 just fine)

Committing a lock file ensures you have consistent environments between developers / CI jobs.

Reply:
Alright, 2 vs. 1 here! I should actually be advocating to include it, reproducibility etc.. Maybe this wouldn't be an issue if we interact via a container anyway.